doc/source/guide/data_types/maps.rst

```diff
@@ -45,10 +45,10 @@ To do this you need to provide `~sunpy.map.map_factory.MapFactory` with both the
 well as some basic meta information. If no header is given then some default
 values as assumed. Here is a simple example::
 
-    import numpy as np
-    data = np.arange(0,100).reshape(10,10)
-    header = {'cdelt1': 10, 'cdelt2': 10, 'telescop':'sunpy'}
-    my_map = sunpy.map.Map(data, header)
+    >>> import numpy as np
+    >>> data = np.arange(0,100).reshape(10,10)
+    >>> header = {'cdelt1': 10, 'cdelt2': 10, 'telescope':'sunpy'}
```

Comment:
mapbase.py defines the observatory property based on the FITS keyword "telescop" - no 'e' since the FITS standard allows for 8 character keywords only.  For consistency with many data sets, perhaps the example above should reflect that?  You can still get the value of the telescope keyword from my_map.meta.get('telescope'). 


    @property
    def observatory(self):
        """Observatory or Telescope name"""
        return self.meta.get('obsrvtry', self.meta.get('telescop', "")).replace("_", " ")